scripts/create_engine_test_assets.py

```diff
@@ -17,6 +17,7 @@ def main():
         (PACKAGES_PATH / "demo_package.py", "demo-package.zip"),
         (PACKAGES_PATH / "configurable_hello_world.py", "configurable_hello_world.zip"),
         (PLUGINS_PATH / "blockifiers" / "blockifier.py", "dummy_blockifier.zip"),
+        (PACKAGES_PATH / "bad_package.py", "bad-package.zip"),
```

Comment:
oh nice. @douglas-reid could use this too to test deploying something that shouldn't work

Reply:
I think @douglas-reid created it - I just committed the test asset so it could be rebuilt :)